docs/4.0/examples/offcanvas/offcanvas.css

```diff
@@ -16,14 +16,15 @@ body {
     padding-right: 1rem;
     padding-left: 1rem;
     overflow-y: auto;
+    visibility: hidden;
     background-color: var(--gray-dark);
-    transition: -webkit-transform .3s ease-in-out;
-    transition: transform .3s ease-in-out;
-    transition: transform .3s ease-in-out, -webkit-transform .3s ease-in-out;
-    -webkit-transform: translateX(100%);
+    transition: visibility .3s ease-in-out, -webkit-transform .3s ease-in-out;
+    transition: visibility .3s ease-in-out, transform .3s ease-in-out;
+    transition: visibility .3s ease-in-out, transform .3s ease-in-out, -webkit-transform .3s ease-in-out;    -webkit-transform: translateX(100%);
```

Comment:
needs a line break here after `;` to pass linting

Reply:
apart from that looking good, and i learned something new here (thought visibility couldn't be transitioned)

Reply:
Oh, sorry. I fixed it.